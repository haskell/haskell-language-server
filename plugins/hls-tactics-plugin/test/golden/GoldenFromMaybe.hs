fromMaybe :: a -> Maybe a -> a
fromMaybe = _
