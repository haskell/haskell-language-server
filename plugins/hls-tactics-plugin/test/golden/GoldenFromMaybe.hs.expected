fromMaybe :: a -> Maybe a -> a
fromMaybe a Nothing = a
fromMaybe _ (Just a') = a'
